integration_tests/test_ibc_rly.py

```
@@ -95,13 +117,8 @@ def distribute_fee(receiver, fee):
     }
 
 
-def fungible(dst, src, amt, denom):
-    return {
-        "receiver": dst,
-        "sender": src,
-        "denom": denom,
-        "amount": amt,
-    }
+def fungible(dst, src, amt, denom, trace=[]):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)